customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/service/SendPasswordImpl.java

```diff
@@ -0,0 +1,42 @@
+package se.inera.intyg.cts.infrastructure.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.cts.domain.model.Termination;
+import se.inera.intyg.cts.domain.repository.TerminationRepository;
+import se.inera.intyg.cts.domain.service.SendPassword;
+import se.inera.intyg.cts.infrastructure.integration.SendSMS;
+import se.inera.intyg.cts.infrastructure.integration.tellustalk.dto.SMSResponseDTO;
+
+@Service
+public class SendPasswordImpl implements SendPassword {
```

Comment:
This class could be named something like SendPasswordWithSMS or something similar. As it is an implementation of SendPassword that sends it through sms.